bigbluebutton-html5/imports/ui/components/app/component.jsx

```diff
@@ -258,7 +258,7 @@ class App extends Component {
         <PollingContainer />
         <ModalContainer />
         <AudioContainer />
-        <ToastContainer />
+        <ToastContainer rtl />
```

Comment:
Why is `rtl` needed here for this container? Isn't `document.body.parentNode.setAttribute('dir', 'rtl');` sufficient?

Reply:
I just followed the docs https://github.com/fkhadra/react-toastify#right-to-left-support